sos/report/plugins/ceph_osd.py

```diff
@@ -39,9 +39,60 @@ def setup(self):
 
         self.add_cmd_output([
             "ceph-disk list",
-            "ceph-volume lvm list",
+            "ceph-volume lvm list"
         ])
 
+        ceph_cmds = [
+           "bluestore bluefs available",
+           "config diff",
+           "config show",
+           "dump_blacklist",
+           "dump_blocked_ops",
+           "dump_historic_ops_by_duration",
+           "dump_historic_slow_ops",
+           "dump_mempools",
+           "dump_ops_in_flight",
+           "dump_op_pq_state",
+           "dump_osd_network",
+           "dump_reservations",
+           "dump_watchers",
+           "log dump",
+           "perf dump",
+           "perf histogram dump",
+           "objecter_requests",
+           "ops",
+           "status",
+           "version",
+        ]
+
+        osd_ids = []
+        out = self.exec_cmd('ps -u ceph -o args')
+
+        if out['status'] == 0:
+            # Get the running OSD process IDs from ps
+            osd_ids = [
+                'osd.%s' % m.split()[5] for m in out['output'].splitlines()
+                if 'ceph --id' in m and len(m.split()) > 5
+                and m.split()[5].isdigit() and m.split()[4] == "--id"
+            ]
+        else:
+            osd_ids = []
```

Comment:
Dead code due to "`osd_ids = []`" in line 68 :)

Reply:
removed, thanks.